src/PopularTag.cpp

```diff
@@ -27,8 +27,8 @@ void PopularTag::increaseCount() {
 
 // overloaded < operator
 bool PopularTag::operator<(const PopularTag& pt) {
-    // check if pt is more popular, has bigger count (meaning User has more Watchables in is
-    //  hisorty containing this tag)
+    // check if pt is more popular, has bigger count (meaning User has more Watchables in his hisorty
```

Comment:
History*, And maybe also it's instead of his because this is a user object and thus inanimate

Reply:
actually it talks about the user as a persion :sweat_smile: 
fixed the grammar problem